src/Kucipong/Handler/Store/Coupon.hs

```diff
@@ -366,6 +372,31 @@ couponPost = do
           , "defaultImage"
           ])
 
+-- | Return the store create page for an admin.
```

Comment:
Though it's trivial, this seems to be "Return the store delete page for an admin.".